tasks/prime.py

```diff
@@ -215,6 +216,65 @@ class PrimeTasks(PrimeDataStorageMixin, ParserTaskMixin, CertTaskMixin, TaskSet)
     tags where appropriate to make filtering for custom tests easier.
     """
 
+    def __init__(self, parent):
+        self.csrf_token = None
+        self.session_token = None
+        super().__init__(parent)
+
+    def customer_path(self, url: str) -> str:
+        return f"{self.user.alternative_host}{url}"
+
+    def on_start(self):
+        self.client.get(self.customer_path("/devlocal-auth/login"))
+        self.csrf_token = self.client.cookies.get("masked_gorilla_csrf")
+        self.client.headers.update({"x-csrf-token": self.csrf_token})
+        resp = self.client.post(
+            self.customer_path("/devlocal-auth/create"),
+            data={"userType": "milmove", "gorilla.csrf.Token": self.csrf_token},
+        )
+        self.session_token = self.client.cookies.get("mil_session_token")
+        if resp.status_code != 200:
+            self.interrupt()
+
+        logged_in_user = self.client.get(self.customer_path("/internal/users/logged_in"))
+        json_resp = logged_in_user.json()
+        service_member_id = json_resp["service_member"]["id"]
+        email = json_resp["email"]
+        user_id = json_resp["id"]
+
+        origin_duty_stations = self.client.get(self.customer_path("/internal/duty_stations?search=29"))
```

Comment:
For the prime it won't matter what origin duty station is selected but it might be good to document which one is actually being selected first in the list when it comes to the office GBLOC and whether the move goes to services counseling or not.

Reply:
The duty station chose is `MCAGCC Twentynine Palms`